cmake/Macros.cmake

```diff
@@ -35,6 +35,17 @@ function(sfml_set_stdlib target)
             message(FATAL_ERROR "Clang is the only supported compiler on macOS")
         endif()
     endif()
+
+    # on some platforms (e.g. ARM), GCC requires linking libatomic to use <atomic> features
+    if(SFML_COMPILER_GCC)
+        include(CheckCXXSourceCompiles)
+        check_cxx_source_compiles("#include <atomic>
+            #include <cstdint>
+            int main(){std::atomic<std::uint64_t> x(1); return x.fetch_add(1);}" ATOMIC_TEST)
+        if(NOT ATOMIC_TEST)
+            target_link_libraries(${target} PUBLIC "-latomic")
+        endif()
+    endif()
```

Comment:
I frequently use GCC on ARM and have never seen this before. What versions of GCC on what platforms are giving you issue?

Reply:
[Our own armhf worker](https://ci.sfml-dev.org/#/builders/29/builds/1864/steps/5/logs/stdio).